dist/js-yaml.js

```diff
@@ -3029,7 +3029,7 @@
   // Return true if the char is printable or a tab.
   // derived from nb-char - #x85 - #xA0 - #x2028 - #x2029.
   function isPrintableWithTabs(c) {
-    return isPrintable(c) || c === CHAR_TAB || c === CHAR_LINE_FEED || c === CHAR_CARRIAGE_RETURN;
```

Comment:
Well would you look at that 🤣. Good find!

Reply:
I wrote this, so it was easy to zero right in on it :)